tests/Integration/RestAPI/ContentHelper/ContentHelperFeatureTestTrait.php

```diff
@@ -0,0 +1,293 @@
+<?php
+/**
+ * Trait for testing the Content Helper API feature.
+ *
+ * @package Parsely
+ * @since   3.17.0
+ */
+
+namespace Parsely\Tests\Integration\RestAPI\ContentHelper;
+
+use Parsely\Permissions;
+use Parsely\REST_API\Base_Endpoint;
+use Parsely\Tests\Integration\TestCase;
+use WP_Error;
+use WP_REST_Request;
+
+/**
+ * Trait for testing the Content Helper API feature.
+ *
+ * @since 3.17.0
+ *
+ * @covers \Parsely\REST_API\Content_Helper\Content_Helper_Feature
+ */
+trait ContentHelperFeatureTestTrait {
+
+
+	/**
+	 * Test that the endpoint is available to the current user.
```

Comment:
It seems that most test function headers are in imperative, though they should be in third person per WP commenting guidelines.